lib/sacastats_web/templates/character/general.html.heex

```diff
@@ -0,0 +1,84 @@
+<link rel="stylesheet" href={Routes.static_path(@conn, "/css/character/general.css")}/>
+
+<div id="characterGeneral" class="row">
+    <div id="characterModelHolder" class="col-md-4 d-none d-md-flex">
+        <figure id="characterModel" class="w-100"></figure>
+        <!-- <img src={Routes.static_path(@conn, "/images/logo_icon.ico")} alt="Sacagawea Icon"> -->
+    </div>
+    <div class="col-md-8 col-12">
+        <div class="stat-section">
+            <h1 id="characterName"><%= @character["response"]["name"]["first"] %></h1>
+            <div class="d-none d-sm-flex float-sm-right">
+                <%= if Integer.parse(@character["response"]["online_status"]) > {0,""} do %>
```

Comment:
Could you change this to be similar to what you did with `next_rank` in the controller? I.e. parse `online_status` in the controller via a pipeline and pass it as its own field in the `character` assign.

Reply:
So is it okay to have a massive controller class? I know that in Springboot you really want to keep your controller classes small.

Reply:
I think right now the size isn't too bad. Once all of the stat pages are in place, then yeah we might want a separate module of helper classes to make things more concise in the controllers, or even split CharacterController into smaller controllers?

Reply:
Not sure how to update the changes here, but they should be pushed.